corehq/apps/domain/forms.py

```diff
@@ -2568,15 +2569,24 @@ def _buttons():
 
 class CreateManageReleasesByAppProfileForm(BaseManageReleasesByAppProfileForm):
     build_profile_id = forms.CharField(label=ugettext_lazy('Build Profile'),
-                                       required=False, widget=Select(choices=[]))
+                                       required=True, widget=SelectMultiple(choices=[]),)
 
     def save(self):
-        try:
-            LatestEnabledBuildProfiles.update_status(self.build, self.cleaned_data['build_profile_id'],
-                                                     active=True)
-        except ValidationError as e:
-            return False, ','.join(e.messages)
-        return True, None
+        success_messages = []
+        error_messages = []
+        for build_profile_id in self.cleaned_data['build_profile_id']:
+            try:
+                LatestEnabledBuildProfiles.update_status(self.build, build_profile_id,
+                                                         active=True)
+                success_messages.append(_('Restriction for profile {profile} set successfully.').format(
+                    profile=self.build.build_profiles[build_profile_id]['name'],
+                ))
+            except ValidationError as e:
+                error_messages.append(_('Restriction for profile {profile} failed: {message}').format(
+                    profile=self.build.build_profiles[build_profile_id]['name'],
+                    message=', '.join(e.messages)
+                ))
+        return error_messages, success_messages
```

Comment:
Nitpick: how about a "No changes made" message if no build profile ids were selected? It's good for users to get some kind of feedback.

Reply:
`build profile id` will always be present, it's a required field now.